### PR TITLE
Update burma.txt

### PR DESCRIPTION
**Pull request by DoktorKure**
fixed federal state option since it wasn't releasing all the possible vassals

### CODE REVIEW

File: CWE/events/burma.txt [hunk 1]
```diff
@@ -174,6 +174,22 @@ country_event = {
 				sphere_owner = { diplomatic_influence = { who = RAK value = 200 } }
 			}
 		}
+		random_owned = {
+			limit = { owner = { is_possible_vassal = MOS } }
+			owner = {
+				release_vassal = MOS
```
Comment (settintotrieste) [inline CWE/events/burma.txt:180]: I believe MOS is a typo

Comment (DoktorKure) [reply CWE/events/burma.txt:180]: It's not, that's the actual tag for Mon State
`1343=
{
	name="Moulmein"
	owner="BUR"
	controller="BUR"
	core="BUR"
	core="MOS"
	garrison=100.000`

File: CWE/events/burma.txt [hunk 2]
```diff
@@ -174,6 +174,22 @@ country_event = {
...
+				RAK = { government = sar_government country_event = 800054 ruling_party_ideology = conservative set_country_flag = former_devolved_nation }
```
Comment (settintotrieste) [inline CWE/events/burma.txt:181]: This is probably the typo then

File: CWE/events/burma.txt [hunk 3]
```diff
@@ -174,6 +174,22 @@ country_event = {
...
+				sphere_owner = { diplomatic_influence = { who = MOS value = 200 } }
+			}
+		}
+		random_owned = {
+			limit = { owner = { is_possible_vassal = KAC } }
+			owner = {
+				release_vassal = KAC
+				RAK = { government = sar_government country_event = 800054 ruling_party_ideology = conservative set_country_flag = former_devolved_nation }
```
Comment (settintotrieste) [inline CWE/events/burma.txt:189]: Same for this one

**Pull request merged**